.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: xenial
 language: go
```

Comment:
we should probably compile it in a docker. We need dapper for ndm probably in future PR.